apiserver/Makefile

```diff
@@ -55,6 +55,11 @@ VERSION_FLAGS = -X $(PACKAGE_NAME)/cmd/apiserver/server.VERSION=$(APISERVER_VERS
 
 include ../lib.Makefile
 
+# TODO Add s390x back to VALIDARCHES. Excluding it should be temporary since
+# there are currently issues in our pipeline with building s390x images. Remove
+# this command once the s390x build pipeline has been fixed.
+VALIDARCHES=amd64 arm64 ppc64le
```

Comment:
s390x support has been added in https://github.com/projectcalico/calico/pull/5793, https://github.com/projectcalico/calico/blob/release-v3.25/apiserver/docker-image/Dockerfile.s390x

Reply:
Unfortunately there were some build issues with our containerized builds that need to be addressed before the s390x artifacts can be built properly for release. See the tests in https://github.com/projectcalico/go-build/pull/413 for more details.